src/constants.rs

```diff
@@ -24,4 +25,6 @@ pub const FALLBACK_DB_URL: &str = "postgres://localhost:5432/agartex-db";
 lazy_static! {
     pub static ref SERVER_URL: SocketAddr = load_env_or_default("SERVER_URL", SocketAddr::new(IpAddr::V4(Ipv4Addr::new(0, 0, 0, 0)), 3200));
     pub static ref SESSION_COOKIE_NAME: String = load_env_or_default("SESSION_COOKIE_NAME", String::from("RSESSID"));
+    pub static ref XUSERID_HEADER_NAME: HeaderName = HeaderName::from_static("x-user-id");
```

Comment:
This doesn't look like it has to be lazy static, since `HeaderName::from_static` is a `const fn`. Either make this just a normal `const XUSERID_HEADER_NAME` or make it look at an env var and only default to "X-User-Id" when it has to.

Reply:
that's true, this variable doesn't need to be `lazy static`. I changed it to `static` as it can't be `constant`  as `XUserId ` cant use it in that situation while implementing `Header` trait